### PR TITLE
Create the kyverno policies to the community that use generate

### PR DESCRIPTION
**Pull request by gparvin**
The Kyverno generate policies are a unique set of policies that can
fill gaps around `NetworkPolicy`, quotas and syncing secrets in a
unique and valuable way.  We would like these policies to become
stable, but first, let's add them to the community.

Note: my previous pr was accidentally wiped out when I deleted my fork.
See the details in that PR:
- https://github.com/stolostron/policy-collection/pull/228

Signed-off-by: Gus Parvin <gparvin@redhat.com>

### CODE REVIEW

File: community/CM-Configuration-Management/policy-kyverno-add-network-policy.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-kyverno-add-network-policy
+  annotations:
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+    policy.open-cluster-management.io/categories: CM Configuration Management
+    policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+spec:
+  remediationAction: enforce
```
Comment (dhaiducek) [inline community/CM-Configuration-Management/policy-kyverno-add-network-policy.yaml]: We should not be enforcing policy-collection policies

File: community/CM-Configuration-Management/policy-kyverno-add-quota.yaml [hunk 2]
```diff
@@ -0,0 +1,101 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-kyverno-add-quota
+  annotations:
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+    policy.open-cluster-management.io/categories: CM Configuration Management
+    policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+spec:
+  remediationAction: enforce
```
Comment (dhaiducek) [inline community/CM-Configuration-Management/policy-kyverno-add-quota.yaml]: We should not be enforcing policy-collection policies

File: community/CM-Configuration-Management/policy-kyverno-sync-secrets.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-kyverno-sync-secrets
+  annotations:
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+    policy.open-cluster-management.io/categories: CM Configuration Management
+    policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+spec:
+  remediationAction: enforce
```
Comment (dhaiducek) [inline community/CM-Configuration-Management/policy-kyverno-sync-secrets.yaml]: We should not be enforcing policy-collection policies

**Review by dhaiducek — CHANGES_REQUESTED** [review]
These are great additions, Gus! Minor change requested.

**Comment by gparvin** [comment]
@dhaiducek Thanks for taking a look.  I addressed your comments and also added comments to the top of each policy.  Please take another look when you have a chance.

**Review by dhaiducek — APPROVED** [review]
Thanks! Good idea providing a reference to the source policy. This is one of those scenarios where it might be nice to give the generator a URL to generate the ACM policy from:
- https://github.com/stolostron/backlog/issues/18204

(Though that may not be as secure/GitOps-y as we'd like)

**Pull request merged**